Orange/canvas/application/workflows/__init__.py

```diff
@@ -32,55 +30,57 @@ def default_entry_point():
     return ep
 
 
-def tutorial_entry_points():
-    """Return an iterator over all tutorials.
+def workflow_entry_points():
+    """Return an iterator over all example workflows.
     """
     default = default_entry_point()
     return chain([default],
-                 pkg_resources.iter_entry_points("orange.widgets.tutorials"))
+                 pkg_resources.iter_entry_points("orange.widgets.workflows"))
```

Comment:
I suggest we use both entry points for an intermediate time, while add-ons can be updated and start using the new entry point.
Later, when everything is up to date, we can remove the 'tutorials' entry point.

I.e. for now we use:
```
    return chain([default],
                 pkg_resources.iter_entry_points("orange.widgets.tutorials"),
                 pkg_resources.iter_entry_points("orange.widgets.workflows"))
```